sonic-thermalctld/scripts/thermalctld

```diff
@@ -624,8 +624,8 @@ class ThermalControlDaemon(DaemonBase):
         """
         logger.log_info("Starting up...")
 
-        import sonic_platform.platform
-        chassis = sonic_platform.platform.Platform().get_chassis()
+        import sonic_platform_base.platform_base
+        chassis = sonic_platform_base.platform_base.PlatformBase().get_chassis()
```

Comment:
This appears to be incorrect. The base class is abstract, and should not be instantiated. The daemon should instantiate the platform-specific concrete Platform class in the sonic_platform package.

Reply:
@jleveque i agree with you. The base class should not be instantiated. A concrete sonic_platform package is required.